src/View/Form/EntityContext.php

```diff
@@ -230,6 +251,26 @@ public function val($field)
     }
 
     /**
+     * Get default value from table schema for given entity field.
+     *
+     * @param string $field Field name.
+     * @param \Cake\Datasource\EntityInterface $entity The entity.
+     * @return mixed
+     */
+    protected function _schemaDefault($field, $entity)
+    {
+        $table = $this->_getTable($entity);
+        if ($table === false) {
+            return null;
+        }
+        $defaults = $table->schema()->defaultValues();
```

Comment:
I was waiting for the `Schema/Table` class name changes to land. After that I can add a new method and update this method to use `defultValue($columns)` instead of `defaultValues()` later on.